packages/backend/src/webhook/service.ts

```diff
@@ -308,29 +312,36 @@ export function finalizeWebhookRecipients(
     tenantIdSet.add(config.operatorTenantId)
   }
 
+  const buildPosRecipient = (): Pick<
+    Webhook,
+    'recipientTenantId' | 'metadata'
+  >[] => {
+    if (!config.posWebhookServiceUrl) {
+      logger?.warn(
+        'Could not create webhook recipient for point of sale service'
+      )
+      return []
+    }
+    return [
+      {
+        recipientTenantId: config.operatorTenantId,
+        metadata: { sendToPosService: true }
+      }
+    ]
+  }
+
+  if (onlyPos) {
+    return buildPosRecipient()
+  }
+
   let recipients: Pick<Webhook, 'recipientTenantId' | 'metadata'>[] = [
     ...tenantIdSet.values()
   ].map((tenantId) => ({
     recipientTenantId: tenantId
   }))
 
-  if (
-    initiationReason === IncomingPaymentInitiationReason.Card &&
-    config.posWebhookServiceUrl
-  ) {
-    recipients = recipients.concat([
-      {
-        recipientTenantId: config.operatorTenantId,
-        metadata: {
-          sendToPosService: true
-        }
-      }
-    ])
-  } else if (
-    initiationReason === IncomingPaymentInitiationReason.Card &&
-    !config.posWebhookServiceUrl
-  ) {
-    logger?.warn('Could not create webhook recipient for point of sale service')
+  if (initiationReason === IncomingPaymentInitiationReason.Card) {
+    recipients = recipients.concat(buildPosRecipient())
   }
```

Comment:
These new outgoing payment events should be sent only to card service, and not POS service (outgoing payments -> card service, incoming payments -> POS Service).

I think we should also make this function take in an object instead of a list of params for readability